common/lib/xmodule/xmodule/course_module.py

```diff
@@ -1438,7 +1408,7 @@ def forum_posts_allowed(self):
 
     @property
     def number(self):
```

Comment:
I'll improve the docstring. What do you mean by "changing the helper method number_for_course_location to be more representative of what a course "number" really is" ?